src/supervisor-api.ts

```diff
@@ -12,7 +12,7 @@ import { checkTruthy } from './lib/validation';
 
 import log from './lib/supervisor-console';
 
-function getKeyFromReq(req: express.Request): string | null {
+function getKeyFromReq(req: express.Request): string | undefined {
```

Comment:
The return value is only used in 1 place in the codebase and it just checks if it's truthy before comparing to another string so switching to undefined will result in the same behavior with the benefit of being able to use match?.[1] which returns undefined if match is falsey.